context/brand-context/springernature/scss/60-utilities/buttons.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Button utilities
+ * These are for styled branded buttons
+ */
+
+// Default button
+
+.u-button {
+	border: $context--button-border-width-style transparent;
+	box-shadow: 0 0 0 1px $context--button-brand-color;
+
+	&:focus {
+		box-shadow: none;
+		outline: none;
+	}
+}
+
+// Button themes
+
+.u-button--primary {
+	@include button-primary
+}
+
+.u-button--secondary {
+	@include button-secondary;
+
+	&:hover {
+		box-shadow: none;
+		border: $context--button-border-width-style $context--button-brand-color;
+	}
+}
+
+// Button variants
+
+.u-button--small {
+	padding: spacing(8) spacing(16);
+}
```

Comment:
I can see why we need to add `u-button--small` here, but the other classes are overriding the utilities that already exist in [the default brand](https://github.com/springernature/frontend-toolkits/blob/b52ed5ddcfea4bc6eeab93516daada684d99fd61/context/brand-context/default/scss/60-utilities/buttons.scss) these options should really be configured in the button mixin so that you can use the classes already defined.